spec/services/dispatch_steps/create_materials_step_spec.rb

```diff
@@ -37,144 +37,157 @@ def make_labware(has_ids)
   def make_submission(labware_bio_ids)
     labwares = labware_bio_ids.map { |bio_ids| make_labware(bio_ids) }
     contact = double('contact', email: 'testing@email')
-    @submission = double(:material_submission, labwares: labwares, contact: contact)
+    owner = double('contact', email: 'contact@email')
```

Comment:
I think it's worth adding some comments to clarify what the difference between contact (in this case Sample Guardian) and owner (in this case material submitter) is, because to someone unfamiliar I imagine it'd be really confusing - especially as they're swapped around and/or replaced with other terms sometimes.